app/src/components/pages/collectionsPage/collectionsPage.tsx

```diff
@@ -1,17 +1,113 @@
 "use client";
 import PageLayout from "../../pageLayout/pageLayout";
-import React from "react";
-import SearchResults from "../../search/results";
+import React, { useState, useRef, useEffect } from "react";
 import {
   Box,
   Heading,
   HorizontalRule,
   SearchBar,
+  Menu,
+  Text,
+  Pagination,
 } from "@nypl/design-system-react-components";
+import { useSearchParams, usePathname, useRouter } from "next/navigation";
 import { headerBreakpoints } from "../../../utils/breakpoints";
+import { CardsGrid } from "../../grids/cardsGrid";
+import LaneLoading from "../../lane/laneLoading";
+import { displayResults, totalNumPages } from "../../../utils/utils";
+import type { SyntheticEvent } from "react";
 import { createAdobeAnalyticsPageName } from "@/src/utils/utils";
 
 export const CollectionsPage = ({ data }) => {
+  const { push } = useRouter();
+  const searchParams = useSearchParams();
+  const pathname = usePathname();
+  const [isLoaded, setIsLoaded] = useState(false);
+
+  const collections = data.collection; // typeof data.collection === "object" ? [data.collection] : data.collection // this breaks empty search
```

Comment:
We talked about this earlier. Either check if `data.collections` is an array, or perhaps check if `data.collections.apiUri` exists. If it's true, then it's a single object which needs to be an array.

Reply:
should be updated now